### ISSUE TITLE
[Research] My VA - Payments Information & On-site Notification: Create Research Plan

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#33295]
## Background
While we are requesting a Midpoint review, we should start the research plan and conversation guide for adding a new Payment Information section to My VA dashboard, as well as adding an On-site Notification for new debt information.

[Payment Information - Project Folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/my-va/payment-history)
[On-site Notifications - Project Folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/notifications/vagov-notifications/mvp)

## Tasks
- [ ] Create a draft of the research plan for the new Payment Information section on My VA
- [ ] Create a draft of the research plan for the new On-site Notification on My VA
- [ ] Send to Samara for review

## Acceptance Criteria
- [ ] A draft research plan is available for review and use in Veteran User Research sessions for these initiatives.

**Comment by andaleliz** [comment]
[Research plan draft](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/discovery-and-research/user-research/payment-information-research-plan.md) started

**Comment by andaleliz** [comment]
@tressaellen Thanks for getting that draft started! I updated it to match the latest OCTO research template, which prompted me to reorganize the research questions a bit. I think we need to add some about onsite notifications but I'm not exactly sure what.

I have a few questions about the research questions you've already added - we can chat vs type about it if that's easier.

> Where do users expect to see payment information on the site?

Is this trying to understand whether or not they would go to My VA to see information about payments they need to make or have received? This will help me figure out if we need to start them on the homepage w/ a broad "show me where you'd find..." question.

> Do users expect to see debt information in the payment section or as an onsite notification?

Can you clarify this one a bit? I'm not sure what it's getting at because there's debt info in both parts of the mock-ups (new debt + total outstanding debts)

>How long do you expect to see that you've been paid the most recent payment?

Just want to confirm this is trying to figure out how long we need to leave a "You received a payment..." notification up.

And last but not least - any thoughts on if you wanted to test desktop or mobile?

**Comment by hdjustice** [comment]
updated ticket to reflect Research plan tasks (conversation guide tasks are in another ticket) and added On-site Notification scope, as the research and conversation guide efforts will be combined to include both projects.

**Comment by tressaellen** [comment]
Hey @andaleliz - I'm sorry I ran out of time yesterday to clean up/finish the research questions! But I'm hoping to get it done before our meeting this afternoon so we can chat through everything together. But thank you for the questions! I'll be sure to address those as I'm going through.

**Comment by andaleliz** [comment]
Oh thank you @tressaellen! I didn't realize you were still going to add more to it. I'll stay tuned.

**Comment by andaleliz** [comment]
@Samara-Strauss I'm writing up the participant criteria for this research plan, and wondering what your thoughts are on having @tmitchellgcio build a prototype in code vs having me build a UX Pin prototype for this research. 

Since My VA is already coded out, it would likely be a matter of copying and pasting a section, updating copy + styling the new alert for testing.  Taylor estimates it would take about a half a day to build that and test on desktop and mobile, providing we don't actually need to wire up any APIs.  The benefit of doing this would be the ability to be more inclusive in recruiting, because people could be on desktop OR mobile, and use any assistive technology we normally have to exclude when testing with UX Pin.

Totally understand if it doesn't make sense to dedicate dev resources to this but figured it be worth checking.  Thanks!

**Comment by Samara-Strauss** [comment]
That sounds great to me if we really anticipate it'll be a low FE lift and if we can accommodate all the different test/use cases we need in the prototype. Can we review the plan to make sure all of our research questions can easily be accounted for in a coded prototype?

**Comment by tmitchellgcio** [comment]
@Samara-Strauss my only concern would be, does this need to be on staging? If so then we may need to write in things that prevent it from being shown or deployed in prod like a feature flipper. If having to build that it would take longer. I'm not as familiar building that part of it so it would take probably another day for me to get it right and test it.

**Comment by andaleliz** [comment]
Thanks for that extra info @tmitchellgcio. Maybe it's not worth it then, unless it's a good learning opportunity and we have extra time in the sprint.

@Samara-Strauss I will draft the research plan as if we're going forward w/ the coded version, and if we decide not to go that route (because of FE LOE or it just won't answer our research questions), I can add the extra criteria to the recruiting section that we need to add to exclude AT and specify devices. Not a big change.

**Comment by andaleliz** [comment]
@Samara-Strauss, I'm still waiting for Tressa to give this plan a once over, and normally wouldn't ask you to review a draft until I've had more eyes on it. However, in light of the holiday hours/PTO, would you mind taking a look at [the draft ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/discovery-and-research/user-research/payment-information-research-plan.md) and providing any feedback before you head out? I'm most interested in making sure the research goals align with your goals for the research so I can work on the conversation guide.

I'll be sure to take care of any grammar/typos so no need for you to provide feedback on those if you see any along the way.

Thanks!

**Comment by Samara-Strauss** [comment]
OK, so I only really reviewed the research goals section. I think you've covered all of the research goals I had in mind. Roughly speaking, the goals are to:

- Make sure payment and debt information is findable (so, we'll want to include tasks to see if people can find this info instead of landing them right on My VA for testing).
- Make sure we have included the right content and links.
- Make sure the debt notification interaction design meets user expectations and needs, or determine where it does not meet user needs and expectations.

I also want to make some clarifications around language that might influence how we approach the actual convo guide. 

> Is the proposed MVP solution (a single dismissible debt notification displayed at the top of the page) helpful to Veterans?

What does "helpful" mean? Is it findable? Does it prompt action to pay a debt? Something else? I think we need to clarify this to make sure we are evaluating the right things the right way.

> What other notifications do users expect to see on VA.gov?

I don't think we need to evaluate this as a primary goal since we already have some research on this from Tressa's initial discovery . However, it certainly doesn't hurt if this comes up naturally.

Unrelated to the above, we were just on a call with the debt team and they mentioned that we should evaluate whether people care how many debts they have re: the debt notification or if they just care a new debt has been added.  This is because we are showing things grouped in the payment section (eg. "You have 3 outstanding debts"), but just "a new debt has been added" at the top of the page. So, wanted to mention this before I forgot. 

---

I also took a quick look at the recruitment criteria:

> notified about payments received from the VA OR been notified about a VA-related debt (both are OK too).

I don't think whether they have received notifications matter. We should just ask if they currently receive disability, compensation, or education payments from the VA OR if they have an outstanding debt with the VA.

**Comment by Samara-Strauss** [comment]
Oh, and for observers, you can take Tom off the list. He's primarily dedicated to the profile work.

**Comment by Samara-Strauss** [comment]
Also, sorry, I realized I missed these comments earlier:

> my only concern would be, does this need to be on staging? 

Given that we want to evaluate whether people expect to find their payment info on My VA in the first place, it probably would help to have this on staging. 

> If so then we may need to write in things that prevent it from being shown or deployed in prod like a feature flipper. If having to build that it would take longer. I'm not as familiar building that part of it so it would take probably another day for me to get it right and test it.

You'll need to put everything behind a feature flag anyway as you build the feature out on staging, though build does not necessarily need to happen before user testing if the LOE is too much and we determine it would be faster and easier to test with a UXPin prototype.

> I will draft the research plan as if we're going forward w/ the coded version, and if we decide not to go that route (because of FE LOE or it just won't answer our research questions), I can add the extra criteria to the recruiting section that we need to add to exclude AT and specify devices. Not a big change.

Sounds good.

**Comment by andaleliz** [comment]
Thanks @Samara-Strauss! I've updated the plan to reflect the feedback you provided.  One update I want to call out is the update to:
>Is the proposed MVP solution (a single dismissible debt notification displayed at the top of the page) helpful to Veterans?

Now it reads as:
**Does the proposed MVP solution (a single dismissible debt notification displayed at the top of the page) help Veterans to more easily take action on their payments or debts?
- Why would a user want to dismiss this alert?
- What do users expect to happen after an alert has been dismissed?
- Are there any areas where the design does not meet Veteran needs?**

-----

Another point I want to address is the timing. Given PTO, and that I'm running research/synthesis with Christina the first week of January, here's the timeline I'm considering. I haven't heard anything about timing for this so I'm assuming it's ASAP and this lines up with what you were planning for:

- Submit for recruiting on 1/5
- Prototype completed by 1/7 (either coded or UX Pin; my understanding is that it's OK to submit to Lauren w/out things being totally done)
- Pilot on 1/10
- Sessions 1/12-1/14

**Comment by Samara-Strauss** [comment]
All of this sounds good to me, including the timeline to conduct the research. Thanks, Liz!

**Comment by tressaellen** [comment]
Things are looking really great! Great work Liz! I don't have much other feedback, but I'm following along and I'm happy to jump in when needed.

**Comment by andaleliz** [comment]
Wonderful! I'm going to close this out then, and start working on the conversation guide.

**Comment by hdjustice** [comment]
> Why would a user want to dismiss this alert?
> What do users expect to happen after an alert has been dismissed?
> Are there any areas where the design does not meet Veteran needs?**

I just thought of a question we could ask regarding the dismissing of the alert.  I was just trying to think about when the notification appears...

"When would the user expect to see the alert appear, after dismissing it?" (you can re-word)
Meaning, if they dismiss the alert, would the user expect to see the alert when they log in again? or when they receive another new/update debt notice?

**Comment by andaleliz** [comment]
Thanks Heather! I will update the research plan so that's included. Great point!
